### DISCUSSION TITLE
Deployment of the walt.id idp kit

### DISCUSSION THREAD

**Discussion by madipa** [walt-id/.github#68] [Help] [answered]
Hello,

i am currently try to deploy the walt.id idp kit on an server. I want to evaluate if the idp kit for our gaia-x project and if it is suitable for our use case
What are the steps I need take? I am running the docker container but when  I integrate it with keycloak I get a Not Found reponse on the webpage. What am I missing?

**Comment by taminobaumann** [comment]
Hey @madipa, 

thanks for reaching out! 

**Use-Case**
Could you please expand on the use-case you are trying to build?


**Setup**
Tutorial: https://docs.walt.id/v/idpkit/tutorials/login-with-nfts-or-keycloak

Can you please explain in a bit more detail what is not working and how to reproduce the situation you are describing?

Best
Tamino

**Reply by madipa** [reply]
Hi @taminobaumann ,

**Use Case**:
We want to use IOTA DID in combination with keycloak for the login. Also I want to check if we can use the Gaia-X verifiable credentials for that.

**My Setup & Issue**
Currently I deployed the idpkit on a server in an docker container and exposed the Port 8080. Via nginx its reachable over an subdomain. I did some configuration like the external uri and the registration of the clients. The   integration in keycloak worked quit well but when I try to login and get redirected I get this "Not Found" message. 

Thank you & best regards
Marco

**Comment by taminobaumann** [comment, marked as answer]
Hi @madipa, 

thanks for the info! 

**Use Case**
Unfortunately the SSI-Kit, the product used by the IDP-Kit for DID registration, VC signing + verification does not support the recent protocol changes of IOTA yet. Because of this, you can not create or verify VCs that hold a did:iota or a did:iota itself.

Please refer to our [roadmap](https://walt-id.notion.site/fcde1687baab42378b3047d4a22eeaca?v=1140dd17c17b4726a70cc1465d20866d) regarding a timeline for when we will upgrade to the latest IOTA protocol changes. 

**Setup & Issue**

Thanks for the detailed explanation. Our team will have a look, and we will get back to you latest next week.

Best
Tamino
